docs/Reference/RPC/commandsRPC.md

```diff
@@ -125,6 +126,73 @@ Common Maintenance Subcommand Options:
 
 <br>
 
+#### addwifisettings
+
+Configure wireless 802.1x settings of an existing, activated AMT device by passing credentials and certificates directly to AMT rather than through RPS/EA/RPC.
+
+More information on configuring AMT to use 802.1x can be found in [802.1x Configuration](../EA/ieee8021xconfig.md).
+
+##### via Config file
+
+1. Create a new file called `config.yaml`. Copy and paste the template below.
+
+    ```yaml
+    ieee801xConfig:
+      name: 'profileName' # profile name (i.e. friendly name)
+      authenticationMethod: 7 # wifi authentication method
+      encryptionMethod: 4 # wifi encryption method
+      clientCert: ''
+      caCert: ''
+      privateKey: ''
+      ssid: '' # wifi SSID
+      username: "" # 8021x username
+      authenticationProtocol: 0 #8021x profile protocol (e.g. EAP-TLS(0))
+      priority: 1 
+    ```
+
+2. Fill in fields with desired options.
+
+3. Provide the `config.yaml` file using the `-config` flag. 
+
+    === "Linux"
+        ``` bash
+        sudo ./rpc maintenance addwifisettings -config config.yaml
+        ```
+    === "Windows"
+        ```
+        .\rpc maintenance addwifisettings -config config.yaml
+        ```
+
+##### via CLI
+
+Alternatively, provide all options directly in the command line.
```

Comment:
Might mention this is would be used for shared library mode --- and passing secrets via command line is insecure.